2023-10-09T00:23:07.6202144Z ##[group]Run actions/setup-python@v4
2023-10-09T00:23:07.6202392Z with:
2023-10-09T00:23:07.6202602Z   python-version: 3.8
2023-10-09T00:23:07.6202834Z   architecture: x64
2023-10-09T00:23:07.6203040Z   cache: pip
2023-10-09T00:23:07.6203250Z   check-latest: false
2023-10-09T00:23:07.6203577Z   token: ***
2023-10-09T00:23:07.6203793Z   update-environment: true
2023-10-09T00:23:07.6204024Z   allow-prereleases: false
2023-10-09T00:23:07.6204234Z ##[endgroup]
2023-10-09T00:23:07.7936768Z ##[group]Installed versions
2023-10-09T00:23:07.8025804Z Successfully set up CPython (3.8.18)
2023-10-09T00:23:07.8026570Z ##[endgroup]
2023-10-09T00:23:08.5891862Z [command]/opt/hostedtoolcache/Python/3.8.18/x64/bin/pip cache dir
2023-10-09T00:23:10.3018638Z /home/runner/.cache/pip
2023-10-09T00:23:10.6607252Z Received 6245959 of 6245959 (100.0%), 36.8 MBs/sec
2023-10-09T00:23:10.6607950Z Cache Size: ~6 MB (6245959 B)
2023-10-09T00:23:10.6635942Z [command]/usr/bin/tar -xf /home/runner/work/_temp/98499c26-dbd2-4ab6-b252-5e9e0e90e513/cache.tzst -P -C /home/runner/work/GitHub_Stats_A/GitHub_Stats_A --use-compress-program unzstd
2023-10-09T00:23:10.6854511Z Cache restored successfully
2023-10-09T00:23:10.6871352Z Cache restored from key: setup-python-Linux-22.04-Ubuntu-python-3.8.18-pip-60f2a6c0212a9b74a155cd81b58dd07fd8b180833132566260d95f4de42fe218
